Paralayout/UIView+Sizing.swift

```diff
@@ -149,25 +163,4 @@ extension UIView {
         sizeToFit(CGSize(width: width, height: height), constraints: constraints)
     }
 
-    /// Resize the view to fit a given size with insets.
-    /// - parameter size: The size to fit, typically the `superview.bounds`.
-    /// - parameter margins: An inset from the supplied size to use (optional, defaults to `0`).
-    public func wrapToFit(_ size: CGSize, margins: CGFloat = 0) {
```

Comment:
I think these methods ultimately prove more confusing than helpful. The same functionality can easily be achieved by calling `sizeToFit(_:constraints:)` with something like:

```swift
view.sizeToFit(bounds.insetAllSides(by: margin).size, constraints: .wrap)
```

I think that ultimately makes the intention more clear, since it specifies how the margin will be applied and better specifies what "wrapping" means (it's a constraint on the size, nothing to do with `numberOfLines` on labels, etc.).